src/care/forall.h

```diff
@@ -491,15 +488,18 @@ namespace care {
    ////////////////////////////////////////////////////////////////////////////////
    template <typename LB, typename Exec>
    void launch_2D_jagged(Exec /*policy*/, int xstart, int /*xend*/, int const * host_lengths, int ystart, int ylength, const char * fileName, int lineNumber, LB && body) {
-      care::RAJAPlugin::pre_forall_hook(chai::CPU, fileName, lineNumber);
+      ProfilePlugin::setFileName(fileName);
```

Comment:
Did you notice that RAJA::forall is not actually called in this method? That means the RAJA plugins won't actually be called. We need to either manually set the chai execution space or figure out how to use RAJA for 2D kernels.

Reply:
changed it to set execution space manually for now